### ISSUE TITLE
Limiter le nombre de contenus touristiques à afficher dans une fiche détails rando

### ISSUE THREAD

**Issue by bruhnild** [GeotrekCE/Geotrek-rando-widget#147]
Il semblerait qu'un nombre trop important de contenus touristiques sur la fiche détail d'une randonnée entraîne des problèmes de lenteur au chargement de la page du widget 

Voir : 
- https://www.explore-millau.com/activites-nature-loisirs/randonnees-pedestres/les-itineraires-randos-autour-de-millau/?trek=46682
- https://www.explore-millau.com/activites-nature-loisirs/millau-terre-de-trail/grands-causses-espace-trail-les-parcours/?trek=50241
- https://www.explore-millau.com/activites-nature-loisirs/millau-terre-de-trail/grands-causses-espace-trail-les-parcours/?trek=45840
-  https://www.explore-millau.com/activites-nature-loisirs/millau-terre-de-trail/grands-causses-espace-trail-les-parcours/?trek=39080

Il serait bien de prévoir une propriété dans le composant `grw-trek-detail` permettant de limiter le nombre de contenus touristiques maximum (comme c'est le cas sur geotrek-rando)

**Comment by camillemonchicourt** [comment]
Oui, on le fait sur Geotrek-rando, c'était une solution initialement et de contournement mais cela n'a pas vraiment de sens.
C'est quand même problématique d'afficher 50 contenus touristiques si il y en a en fait 78... c'est faux et cela peut apporter des incompréhensions, voir des mécontents.

Au final, il vaut mieux les afficher tous ou aucun mais pas un entre-deux peu compréhensible, faux et source de problèmes...

Dans les usages, on s'est rendu compte que c'était surtout sur les fiches mère des itinérances qu'on pouvait avoir beaucoup beaucoup de contenus liés car les tracés sont longs. Et on a donc privilégier le fait de ne pas du tout afficher les contenus touristiques sur les fiches mère des itinérances. Elles le sont sur les fiches des étapes ou c'est bien plus utile et lisible : https://github.com/GeotrekCE/Geotrek-rando-v3/issues/847

Bon, ce n'est pas parfait non plus comme solution, mais selon moi, afficher une partie des contenus uniquement est la pire des solutions.

**Comment by gbvt** [comment]
On s'est posé la même question au CD06, pour arriver à la même problématique que Camille : pour être "impartial", il faut être exhaustif. Sinon ce serait favoriser un service plutôt qu'un autre sur une base totalement aléatoire.
